inc/migration.class.php

```diff
@@ -976,4 +976,53 @@ public function setContext($context) {
       $this->context = $context;
       return $this;
    }
+
+   /**
+    * Add new right
+    * Give full rights to profiles having config right
+    *
+    * @param string $name Right name
+    *
+    * @return void
+    */
+   public function addRight($name) {
+      global $DB;
+
+      $count = countElementsInTable(
+         'glpi_profilerights',
+         ['name' => $name]
+      );
+      if ($count == 0) {
+         //new right for certificate
+         //give full rights to profiles having config right
+         foreach ($DB->request("glpi_profilerights", ['name' => 'config']) as $profrights) {
+            if ($profrights['rights'] && (READ + UPDATE)) {
+               $rightValue = CREATE | READ | UPDATE | DELETE  | PURGE | READNOTE | UPDATENOTE | UNLOCK;
```

Comment:
globally ok with this pr, but this part seems strange.
Here a list of issue i though about:
- DELETE is present only for itemtype with is_deleted  field
- READNOTE & UPDATE NOTEfor those which have notes tab

CREATE-UPDATE-PURGE-UNLOCK should be ok, 
But more generally some rights have only READ, ex reports, also child itemtypes have less rights, and some rights don't have any standards constants

Reply:
I'm pretty sure we cannot find a really generic way... I've based the method on 2 existing cases from upgrade scripts. Maybe should we define a default standard "rightValue" (which rights for this one?) and permit to pass specific values calling the method?

Reply:
I really do not know if we can have side effects... But if we cans be as generic as possible from the begining, it will be better. I've added a new commit to fix that.